Makefile

```diff
@@ -31,6 +31,7 @@ STEPS := all tests apps clean check info tools %.debug
 HOST := $(shell uname)
 TARGET ?= $(HOST)
 BUILD_TYPE ?= Debug
+DISABLE_DOCKER ?= n
```

Comment:
nit; rename `DISABLE_DOCKER ` to just `DOCKER` and set the default value to `1` so we can use syntax like `make TARGET=Linux DOCKER=0 apps` instead.

Reply:
The variable name `DOCKER` is already used to store the docker executable line 13 : `DOCKER := docker`.

Reply:
Or maybe caller the `docker` executable variable `DOCKER_EXE` and the option deactivate it `DOCKER`, defaulting to `DOCKER=y`.